CONTRIBUTING.md

```diff
@@ -5,23 +5,13 @@
 1. Install Xcode: the easiest is to install it as a [free app from the Mac AppStore](https://itunes.apple.com/us/app/xcode/id497799835?mt=12).
 1. Clone the [repo](github.com/schibsted/account-sdk-ios).
 1. Install required development tools:
-    1. **Option 1:** If you want to install all the tools at once, get [Bundler](https://bundler.io/) and then install with:
+    If you want to install all the tools at once, get [Bundler](https://bundler.io/) and then install with:
 
-            bundler install
-            pod setup --verbose
+        bundler install
 
-        this will install everything with known working versions from the [Gemfile](Gemfile), including [CocoaPods](https://cocoapods.org/).
+    this will install everything with known working versions from the [Gemfile](https://github.com/schibsted/account-sdk-ios/blob/master/Gemfile), including [CocoaPods](https://cocoapods.org/).
 
-        **Note**: Installing a ruby version manager such as [RVM](https://rvm.io) will make life a little easier
-
-    1. **Option 2:** If you want to just be able to build and test with Xcode, install [CocoaPods](https://cocoapods.org/) by running in the terminal:
-
-            sudo gem install cocoapods
-            pod setup --verbose
-
-        If there's a problem in [Podfile.lock](Example/Podfile.lock) having unquoted paths, update psych:
-
-            sudo gem install psych -v 2.2.2
+    **Note**: Installing a ruby version manager such as [RVM](https://rvm.io) will make life a little easier
```

Comment:
But it might break App Store submission 😜

Reply:
Haha 😭